tests/test_html2docx.py

```diff
@@ -33,14 +38,12 @@ def test_html2docx(html_path, spec_path):
         runs_spec = p_spec["runs"]
         assert len(p.runs) == len(runs_spec)
         for run, run_spec in zip(p.runs, runs_spec):
-            assert run.text == run_spec["text"]
-            for attr in (
-                "bold",
-                "italic",
-                "strike",
-                "subscript",
-                "superscript",
-                "underline",
-            ):
-                msg = f"Wrong {attr} for text '{run.text}' in {html_path}"
+            assert run.text == run_spec.pop("text")
+            unknown = set(run_spec).difference(FONT_ATTRS)
+            assert not unknown, "Unknown attributes in {}: {}".format(
+                spec_path.relative_to(PROJECT_DIR), ", ".join(unknown)
+            )
+            for attr in FONT_ATTRS:
+                rel_path = html_path.relative_to(PROJECT_DIR)
```

Comment:
Since this value does not change based on `attr` or `run, run_spec`, the line can be pulled out of the run loop.